reference/api/stats.md

```diff
@@ -1,6 +1,20 @@
 # Stats
 
-`Stats` gives extended information and metrics about indexes and the MeiliSearch database.
+`Stats` gives extended information and metrics about indexes and the MeiliSearch database. It returns the following details:
+
+#### 1. Number of documents
```

Comment:
There's a MarkDown style rule about incrementing headers progressively that we try to follow. Basically we don't want to jump from an H1 to an H4 if we can avoid it. We also probably don't want to use a numbered list if the order doesn't matter.

I wonder what's the best way to present this information. A table could be good, since we could include a column for whether it's returned by "get stats of one index" or by "get stats of all indexes", or both. On the other hand it could look a bit messy.

Perhaps it would be better to emulate [Stripe's approach](https://stripe.com/docs/js/initializing#init_stripe_js-publishableKey) to method parameters. Something like this:

## Fields returned

### Get stats for one index

`GET /indexes/:index_uid/stats` returns the following fields:

#### `numberOfDocuments`

blah blah description blah

#### `isIndexing`

yadda yadda description here

#### `fieldDistribution`

description

### Get stats for all indexes

In addition to all fields returned by [`GET /indexes/:index_uid/stats`](#get-stats-for-one-index), `GET /stats` returns the following instance-level fields:

#### `databaseSize`

description

#### `lastUpdate`

description

What do you think @guimachiavelli ?

Reply:
Sorry, completely missed this notification.

Being extremely nitpicky about this type of thing, I agree with @dichotommy: we shouldn't use ordered lists when the order of the list items is not important and headings should follow a logic progression from `#` to `##` to `###` and so on. Having a clear hierarchy that we always follow can be very useful down the road.

I'm also not a big fan of tables because they can don't degrade well when conditions are not ideal (too much content, too little content, different types and lengths of content in the same row, etc), so I like Stripe's approach. Keep in mind I worked for a long time as a front-end dev, so I'm naturally biased against tables, though.

Stripe's approach degrades nicely and I like it very much, but wouldn't using it here make the stats API reference inconsistent with all other endpoints?

Reply:
Agreed with this being inconsistent with all other endpoints but this is the only one where we need to explain what the returned fields mean.

Reply:
I guess we can just move forward with @dichotommy's suggestion, then 👍

Reply:
I think introducing a little inconsistency is acceptable if we take the decision together, and agree to use this as a model for future API reference revisions.

Reply:
Can we not use the field names as the heading?
I think Number of documents looks better as a heading compared to `numberOfDocuments`

![image](https://user-images.githubusercontent.com/90181761/143026709-6b379b22-8996-499f-a31e-fd2fbea59b11.png)
![image](https://user-images.githubusercontent.com/90181761/143026789-b5492784-3b11-4af7-9f2d-72f138efd35f.png)

I don’t think Is Indexing is a good alternative to `isIndexing`

Reply:
I personally prefer the first screenshot, it feels like I have instant visual clarity that I'm looking at definitions of fields. What do you think @guimachiavelli ?

Reply:
I'm team first screenshot too. I think I prefer it because I have an immediate visual cue that the each heading is describing a key/field, whereas "Number of documents" requires me to read the text to be sure.